sdk/java-sdk/src/main/java/kalix/javasdk/workflowentity/WorkflowEntity.java

```diff
@@ -316,6 +374,13 @@ public CallBuilder(String name,  Function<Input, DeferredCall<DefCallInput, DefC
         this.callFunc = callFunc;
       }
 
+      /**
+       * Transition to a next step based on step action result.
```

Comment:
```suggestion
       * Transition to the next step based on the result of the step action.
```
Maybe?

Reply:
If you agree feel free to change the same one below.

Reply:
thanks